Plots/Panels/NCL_panel_14.py

```diff
@@ -0,0 +1,200 @@
+"""
+NCL_panel_14.py
+===============
+This script illustrates the following concepts:
+   - Combining two sets of paneled plots on one page
+   - Adding a common labelbar to paneled plots
+   - Reversing the Y axis
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: http://www.ncl.ucar.edu/Applications/Scripts/panel_14.ncl
+    - Original NCL plot: http://www.ncl.ucar.edu/Applications/Images/panel_14_lg.png
+"""
+
+##############################################################################
+# Import packages:
+
+from matplotlib.ticker import FixedLocator
+import matplotlib.pyplot as plt
+import xarray as xr
+import numpy as np
+
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+import geocat.viz.util as gvutil
+
+##############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/h_avg_Y0191_D000.00.nc"),
+                     decode_times=False)
+# Ensure longitudes range from 0 to 360 degrees
+T = gvutil.xr_add_cyclic_longitudes(ds.T, "lon_t")
+
+# Extract slices of data for each panel
+T1 = T.isel(time=0).sel(lat_t=30, lon_t=180, method="nearest")
+T2 = T.isel(time=0).sel(lat_t=-30, lon_t=180, method="nearest")
+T3 = T.isel(time=0).sel(lon_t=270, method="nearest")
+T4 = T.isel(time=0).sel(lon_t=200, method="nearest")
+
+##############################################################################
+# Plot:
+fig = plt.figure(figsize=(10, 13))
+
+# Add the subplots
+ax1 = plt.subplot(2, 2, 1)  # upper left cell of grid
```

Comment:
I think using gridspec is a better idea for this subplot due to the extra white space below the plot. 
```suggestion
ax1 = fig.add_subplot(spec[0, 0])  # upper left cell of grid
```